src/flow-spec/IBlockExit.ts

```diff
@@ -24,16 +24,14 @@ export interface IBlockExit {
   /**
    * A globally unique identifier for this Block.  (See UUID Format: https://floip.gitbook.io/flow-specification/flows#uuid-format)
    *
-   * @TJS-pattern ^[0-9a-fA-F]{8}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{12}$
+   * @pattern ^[0-9a-fA-F]{8}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{12}$
    */
   uuid: string
 
   /**
    * This is the human-readable name of the exit (as a translated resource ID), which might be presented to a contact.
-   *
-   * @TJS-pattern ^[0-9a-fA-F]{8}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{12}$
```

Comment:
I removed this as it's not matching with current implementation so far, we're using simple `string` (eg: `Default`, `Error`, etc) rather than `uuid` here.
@bzabos , shall we update the spec description to reflect this usage or shall we change the way we set this `label` in flow-builder ?